compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1841,6 +1841,17 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode
          if ((et == TR::Int8 || et == TR::Int16 || et == TR::Int32 || et == TR::Int64 || et == TR::Float || et == TR::Double))
             return true;
 
+      case OMR::vmin:
+         if (et == TR::Int8 || et == TR::Int16 || et == TR::Int32 || et == TR::Int64 || et == TR::Float || et == TR::Double)
+            return true;
+         else
+            return false;
```

Comment:
`return (et == TR::Int8 || et == TR::Int16 || et == TR::Int32 || et == TR::Int64 || et == TR::Float || et == TR::Double);` is simple and readable as I commented in another PR.

Reply:
For the sake of consistency, should the rest of the vector opcodes also be formatted this way? And as you mentioned in the previous PR, should I add this change now, or do we want to save it for a future PR?

Reply:
You can change it in a separate PR.